### PR TITLE
Example for GitHub action

### PR DESCRIPTION
**Pull request by fabiangabel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,11 +67,33 @@ after_success:
...
+  codecov:
+    runs-on: ubuntu-latest
+    container: perl:latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Run codecov GitHub Action
+        env:
+          CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
+        run: |
+          cpanm --quiet --notest Module::Build Devel::Cover Devel::Cover::Report::Codecov
```
Comment (foursixnine) [inline README.md]: This should be [Devel::Cover::Report::Codecovbash](https://github.com/mudler/Mojo-IOLoop-ReadWriteProcess/pull/27#issuecomment-983682803) as the bash uploader is gonna be deprecated... 
```suggestion
          cpanm --quiet --notest Module::Build Devel::Cover Devel::Cover::Report::Codecovbash
```

**Review by foursixnine — APPROVED** [review]

**Comment by oalders** [comment]
Any reason not to merge this?

**Comment by pine** [comment]
Hello, I’m sorry but I didn’t notice that. I will review it later.

**Comment by pine** [comment]
It seems that the link below is the cause.

https://about.codecov.io/blog/codecov-uploader-deprecation-plan/

**Pull request merged**